src/cli/outdated_command.zig

```diff
@@ -116,22 +116,22 @@ pub const OutdatedCommand = struct {
     // TODO: use in `bun pack, publish, run, ...`
     const FilterType = union(enum) {
         all,
-        name: []const u32,
-        path: []const u32,
+        name: []const u8,
+        path: []const u8,
 
-        pub fn init(pattern: []const u32, is_path: bool) @This() {
+        pub fn init(pattern: []const u8, is_path: bool) @This() {
             return if (is_path) .{
                 .path = pattern,
             } else .{
                 .name = pattern,
             };
         }
 
-        pub fn deinit(this: @This(), allocator: std.mem.Allocator) void {
-            switch (this) {
-                .path, .name => |pattern| allocator.free(pattern),
-                else => {},
-            }
+        pub fn deinit(_: @This(), _: std.mem.Allocator) void {
+            // switch (this) {
```

Comment:
Can you delete this function so that the next person working on this code doesn't think it's a memory leak?